zuul.d/playbooks/atmosphere/run.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2024 VEXXHOST, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+- name: Fetch Atmosphere
+  hosts: all
+  tasks:
+    - name: Git clone Atmosphere
+      become: true
+      ansible.builtin.git:
+        repo: 'https://github.com/vexxhost/atmosphere.git'
+        dest: /tmp/atmosphere
+        version: "{{ lookup('ansible.builtin.env', 'atmosphere_release', default='main') }}"
+    - name: Run Tox to build Atmosphere Environment
+      become: true
+      ansible.builtin.command:
+        tox -e molecule-venv -- converge -s csi
+      args:
+        chdir: /tmp/atmosphere
+
+    # We can drop this part `ceph-osd-exporter=enabled` exists in all testing branches.
+    - name: Add lable to `instance` node
+      become: true
+      ansible.builtin.command:
+        kubectl label nodes instance ceph-osd-exporter=enabled
```

Comment:
i think we need to set node label for `ceph_osd_group` in `vexxhost.atmosphere.csi` playbook which is executed after ceph and k8s installed.
maybe `ceph/role: osd` ?

Reply:
How about we using ceph/role=osd here and adding it to a new atmosphere/roles/node_labels and call it in vexxhost.atmosphere.infrastructure? I think that will reduce confusion in csi as it doesn't need that label anywhere.